lib/ansible/modules/network/avi/avi_errorpagebody.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/python
+#
+# Created on Aug 25, 2016
+# @author: Gaurav Rastogi (grastogi@avinetworks.com)
+#          Eric Anderson (eanderson@avinetworks.com)
+# module_check: supported
+#
+#
+# This file is part of Ansible
```

Comment:
Replace Line 9 to 23 with 

```
# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Reply:
The current license header was provided previously by Ansible Networking Group. I haven't seen any guideline on it. @gundalow any comments?

Reply:
This is documented at http://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#copyright

Reply:
@grastogi23 It has changed since you created your initial modules. Please use the new form

Reply:
@sivel @gundalow Sure. I can change it. Also, would you like me to change it for all existing modules as well?

Reply:
PR already merged which fixes the existing modules